box_it_script/README.md

```diff
@@ -0,0 +1,6 @@
+# HOMEWORK 1 #
```

Comment:
did you branch out from homework 1 branch? this pr shouldn't contain commits from `box_it_script`